assets/js/hooks/hoverune.js

```diff
@@ -37,6 +37,16 @@ function floatHoveRune({ clientX, clientY }) {
   });
 }
 
+const findMatchingSpan = ({ node_id, field }) => {
```

Comment:
perhaps this should be generic `findMatchingDomNode` since very very soon we will have more than just spans to focus on

Reply:
yee something more abstract on the search query

Reply:
kiv to future, okay to not change it for now